irohad/consensus/yac/impl/supermajority_checker_bft.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "consensus/yac/impl/supermajority_checker_bft.hpp"
+
+#include <boost/range/algorithm/max_element.hpp>
+#include <boost/range/numeric.hpp>
+#include "consensus/yac/impl/supermajority_checker_kf1.hpp"
+
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+
+      bool SupermajorityCheckerBft::hasSupermajority(
```

Comment:
This file is almost the same as `supermajority_checker_cft.cpp`, the only change is `k` parameter in `hasSupermajority`. Maybe it is better to replace these two classes with class template, and make `k` a template parameter?